DemoApp/Sources/Views/CallView/DemoChatModifier.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  DemoChatModifier.swift
+//  DemoApp
+//
+//  Created by Ilias Pavlidakis on 10/10/23.
+//
+
+import Foundation
+import SwiftUI
+import StreamVideoSwiftUI
+
+struct ChatModifier: ViewModifier {
```

Comment:
I like this idea of having chat modifier. Maybe we could introduce chat extensions for video as reusable components in the future 👍

Reply:
Sounds a good idea and will make integration much easier for customers. I think we should have that in our roadmap